sharedgo/deletionPicker.go

```diff
@@ -0,0 +1,21 @@
+// Picks what file from 10 to 50 (files 0-9 kept in the system for persistence) to delete
+
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"time"
+)
+
+func pick_file_to_delete() string {
```

Comment:
Before calling delete we will always have 50 files on hand ie it picks a random file and then later we call create file to create file with the exactly same id as the file that was deleted so it can always picka file to delete from the files with ids >=10 and <=50!